scripts/configs/config_7m.yaml

```diff
@@ -1,234 +1,141 @@
 data:
-  # dimensions of training images
+  # dimensions of the raw training images. your computer will tell you WIDTH X HEIGHT
   image_orig_dims:
-    width: 1056
-    height: 1024
-
-  # resize dimensions to streamline model creation
+    width: 1328
+    height: 1048
+  # resize dimensions: we do this internally to accelerate training. make sure its a multiple of 128.
   image_resize_dims:
-    width: 384
-    height: 384
-
-  # ABSOLUTE path to data directory
-  data_dir: /home/zeus/content/data/fish_implant
-  # ABSOLUTE path to unlabeled videos' directory
-  video_dir: /home/zeus/content/data/fish_implant/videos
+    width: 256
+    height: 256
+  # ABSOLUTE path to data directory. the below is just a toy example for running Getting Started scripts
+  data_dir: /mnt/scratch2/farzad/7m/data
+  # ABSOLUTE path to unlabeled videos' directory. the below is just a toy example for running Getting Started scripts
+  video_dir: videos
   # location of labels; for example script, this should be relative to `data_dir`
-  csv_file: master_multicam_labeledFrames.csv
-  # downsample heatmaps - 2 | 3
+  csv_file:
```

Comment:
@danbider what do you think about enumerating all of the actual csv file names vs programmatically assigning them based on a root csv file name and the view names? forces more structure on the data (which people may not like) but also reduces room for error.